src/sentry/api/endpoints/api_tokens.py

```diff
@@ -119,3 +123,33 @@ def delete(self, request: Request):
         analytics.record("api_token.deleted", user_id=request.user.id)
 
         return Response(status=204)
+
+    @method_decorator(never_cache)
+    def put(self, request: Request) -> Response:
+        if "scopes" in request.data:
+            return Response(status=403)
```

Comment:
oh, can we add a message to this saying you cannot edit auth token scopes after creation